evap/results/tests/test_exporters.py

```diff
@@ -590,3 +590,120 @@ def test_text_answer_export(self):
         self.assertEqual(sheet.row_values(3)[0], questions[0].text)
         self.assertEqual(sheet.row_values(5)[0], questions[1].text)
         self.assertEqual(sheet.row_values(6)[0], questions[2].text)
+
+    def test_total_average(self):
+        program = baker.make(Program)
+
+        questionnaire = baker.make(Questionnaire, order=1, type=Questionnaire.Type.TOP)
+
+        question = baker.make(Question, type=QuestionType.POSITIVE_LIKERT, questionnaire=questionnaire)
+
+        evaluation_1 = baker.make(
+            Evaluation,
+            course__programs=[program],
+            state=Evaluation.State.PUBLISHED,
+            _participant_count=2,
+            _voter_count=2,
+        )
+
+        evaluation_2 = baker.make(
+            Evaluation,
+            course__programs=[program],
+            state=Evaluation.State.PUBLISHED,
+            _participant_count=2,
+            _voter_count=2,
+        )
+
+        evaluation_1.general_contribution.questionnaires.set([questionnaire])
+
+        make_rating_answer_counters(question, evaluation_1.general_contribution)
+
+        evaluation_2.general_contribution.questionnaires.set([questionnaire])
+
+        make_rating_answer_counters(question, evaluation_2.general_contribution)
+
+        cache_results(evaluation_1)
+        cache_results(evaluation_2)
+
+        binary_content = BytesIO()
+        ResultsExporter().export(
+            binary_content,
+            [evaluation_1.course.semester, evaluation_2.course.semester],
+            [
+                (
+                    [course_program.id for course_program in evaluation_1.course.programs.all()]
+                    + [course_program.id for course_program in evaluation_2.course.programs.all()],
+                    [evaluation_1.course.type.id, evaluation_2.course.type.id],
+                )
+            ],
+            True,
+            True,
+        )
+        binary_content.seek(0)
+        workbook = xlrd.open_workbook(file_contents=binary_content.read())
+
+        self.assertEqual(
+            float(workbook.sheets()[0].row_values(5)[1]),
+            (float(workbook.sheets()[0].row_values(5)[2]) + float(workbook.sheets()[0].row_values(5)[3])) / 2,
```

Comment:
From a quick glance, it looks like the setup above creates the same exact average result for both evaluations and same count of answers inside of them. This doesn't test whether we compute the average-of-averages with correct weighting. Can we change them to have different answer counts?

Coincidentally, float-equality here made me wonder about correctness. It should work if the two values here are identical, the sum is then 2 * the original value, which is divisible by 2 to perfectly recreate the original value. However, I'd like the test to test a somewhat more sophisticated case. We'll likely need to use `assertAlmostEqual` then.

Reply:
@fidoriel This currently compares a value the system produces against another value that the system produces, which leaves the chance that both might be wrong, but the test still passes.

For a bit more assertiveness, I would appreciate if we could change the assertion here to the exact string that should be shown in the output, optimally in the string-domain (without to-number conversion)